tools/java/org/unicode/cldr/tool/GenerateProductionData.java

```diff
@@ -64,7 +69,7 @@
             .setMatch(".*")),
         destinationDirectory(new Params()
             .setHelp("destination common directory")
-            .setDefault(CLDRPaths.AUX_DIRECTORY + "production/common")
+            .setDefault(CLDRPaths.STAGING_DIRECTORY + "production/common")
```

Comment:
See comment below about cldr-aux vs cldr-staging

Reply:
> Hmmm. It is more convenient to write to the _local_ directory for cldr-staging: one can then easily diff, etc., but doesn't have to check in.
> 
> Moreover, I think it could be useful for collaboration to be able to check in versions for other people to look at on occasion.
> 
> The important thing would be that that the _tags_ be set "officially" for the release alphas, betas, and finals.

OK, makes sense. Need to update the CLDR-to-ICU build instruction then, I guess (since they say GenerateProductionData defaults to cldr-aux)